installation-linux.html

```diff
@@ -55,52 +55,82 @@
 
 <p>This page describes the installation and compilation of Gregorio software under a GNU/Linux system.  Whether you install a precompiled version or whether you compile from source, it's a quick and easy operation.</p>
```

Comment:
This second sentence mentions a "precompiled version", but the page only documents build-from-source.  I think we can just drop the sentence that starts with the word `Whether`.